DocsCode/NavigationTutorial/NavigationTutorialViewController.swift

```diff
@@ -58,7 +58,7 @@ class ViewController: UIViewController, MGLMapViewDelegate {
         // Calcuate the route from the user's location to the set destination
         calculateRoute(from: (mapView.userLocation!.coordinate), to: annotation.coordinate) { (route, error) in
             if error != nil {
-                print("Error calculating route")
+                NSLog("Error calculating route: \(String(describing: error))")
```

Comment:
So `print()` _is_ the recommended way to put text to console in Swift. 😁 It’s possible to use `NSLog()` in Swift (and there are [behavioral differences](https://stackoverflow.com/a/25951564/2094275) that aren’t relevant here), but `print()` is preferred.

The issue with the ObjC code was in using `printf()`, which is a C function.